Makefile.ci

```diff
@@ -266,6 +266,14 @@ ci-platform_full: $(CI_PLATFORM_FULL)
 $(CI_TARGETS): ci-%:
 	+./dev/ci/ci-wrapper.sh $*
 
+# Helper to update all CI repositories already downloaded
+ci-update:
+	+./dev/ci/ci-update.sh
```

Comment:
```suggestion
	./dev/ci/ci-update.sh
```
(+ means the script may call make)

Reply:
Really? From what I gather from the internet with a plus symbol
> the command is executed even if Make is invoked in a "do not execute" mode

Reply:
That and IIRC some jobserver stuff are the effect, but it's meant for recursive make
Do you want these commands to be executed when make is invoked in do not execute mode?

Reply:
This is just a shorthand to quickly make the target, I don't think there is anything deeper to that. I'm going to remove the + then.